crates/core/tedge_api/src/messages.rs

```diff
@@ -534,6 +535,74 @@ impl RestartOperationResponse {
     }
 }
 
+#[derive(Debug, Deserialize, Serialize, PartialEq, Copy, Eq, Clone)]
+#[serde(rename_all = "camelCase")]
+pub enum CommandStatus {
+    Init,
```

Comment:
There is a similar update pushed by @Ruadhri17 : https://github.com/thin-edge/thin-edge.io/pull/2086/files#diff-27b71db1e49e82260fedb815210084f2ed0d2d9c694ad8c17e859707458b3844R11

Reply:
This has been resolved by https://github.com/thin-edge/thin-edge.io/pull/2108/commits/70d6c77edbb0621151c369093724f8a6c4423a52